PF/Assignment1/BinarySearch.java

```diff
@@ -0,0 +1,79 @@
+import java.util.Scanner;
```

Comment:
ACTION: Remove import scanner as you are not taking any input from user in this class.